v63003/u_inref/gtm8787.csh

```diff
@@ -14,40 +14,35 @@
 # Tests that Mupip Journal -Extract=-stdout handles its termination appropriately
 #
 
-# Creating database,journal file
-setenv gtm_test_db_format "NO_CHANGE"
-source $gtm_tst/com/gtm_test_setbgaccess.csh
-source $gtm_tst/com/gtm_test_setbeforeimage.csh
+echo "# Creating database,journal file"
 $gtm_tst/com/dbcreate.csh mumps 1>>& create.out
 if ($status) then
 	echo "DB Create Failed, Output Below"
 	cat create.out
 endif
-$MUPIP Set -Region Default -Journal=enable,on,before,file=mumps.mjl >>& jnlcreate.out
+$MUPIP Set -Region Default -Journal=enable,on,nobefore,file=mumps.mjl >>& jnlcreate.out
 if ($status) then
 	echo "Journal Create Failed, Output Below"
 	cat jnlcreate.out
 endif
 
 # Updating database with massive amount of information so it will take time to extract,
 # allowing the system to easily exit while extracting
-$ydb_dist/mumps -run ^%XCMD "for i=1:1:10 set ^x(i)=i"
+$ydb_dist/mumps -run ^%XCMD "for i=1:1:1000 set ^x(i)=i"
 set t = `date +"%b %e %H:%M:%S"`
 
 echo "# Running extract and terminating in separate terminal"
 (expect -d $gtm_tst/$tst/u_inref/gtm8787.exp > expect.outx) >& xpect.dbg
 if ($status) then
 	echo "EXPECT FAILED"
 endif
-$ydb_dist/mumps -run gtm8787 >>& temp.out
-set s = `cat temp.out`
-$gtm_tst/com/getoper.csh "$t" "" log.outx "" $s
+$gtm_tst/com/getoper.csh "$t"
```

Comment:
I would prefer a comment at the end of the line indicating the syslog messages will be logged to syslog.txt.